WebInkEnhancement/explainer.md

```diff
@@ -148,7 +146,9 @@ We considered a few different locations for where the method `setLastRenderedPoi
 
   This seemed a bit too generic and scoping to a new namespace seemed appropriate.
 
-Instead of a concrete type, perhaps the `PenStrokeStyle` should be more generic in such a way that presenters can describe their capabilities via a dictionary. I'm not quite sure what the most ergonomic way of exposing this would be.
+Due to uncertainty around the correct execution when `setLastRenderedPoint` is called before setting the stroke style, and it being likely that the radius could change frequently, we decided it may be best to require all 3 arguments (event, color, radius) in every call to `setLastRenderedPoint`.

```

Comment:
Nit: change "all 3 arguments" to "all relevant properties of rendering the ink stroke" now that color and radius are not technically arguments any longer